git-archive-all.sh

```diff
@@ -246,8 +248,9 @@ fi
 if [ $VERBOSE -eq 1 ]; then
     echo -n "archiving submodules..."
 fi
+git submodule >>$TMPLIST
```

Comment:
Possibly worth quoting `$TMPLIST` here, just for the sake of taking defensive safety measures.